app/scripts/templates/settings/clients.mustache

```diff
@@ -18,7 +18,7 @@
         {{#clients}}
           <li class="client {{type}} client-{{clientType}} {{#isCurrentDevice}}client-current{{/isCurrentDevice}}"
             data-name="{{name}}" id="{{id}}">
-            <div class="client-name" title="{{name}}">{{name}}</div>
+            <div class="client-name" title="{{name}}" scope="{{scope}}">{{name}}</div>
```

Comment:
We shy away from using non-standard DOM attribute names, instead [`data-` attributes](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes) are used to store data on the DOM.